### PR TITLE
use more inclusive language in our documentation

### PR DESCRIPTION
**Pull request by atomfrede**
Today I found this article on dev https://dev.to/meeshkan/how-to-remove-condescending-language-from-documentation-4a5p
I was not aware of [some tooling](https://alexjs.com/) to help to identify texts, that might not be inclusive. Overall we have 279 warnings most of them are e.g. using easy or just which might be insensitive. If we all agree I would try to remove most warnings. We could try to integrate alex into the build process to raise awareness for condescending language.

### CODE REVIEW

File: pages/setting_up_ci.md [hunk 1]
```diff
@@ -26,6 +26,7 @@ JHipster should support the following CI systems out of the box:
 - Travis: refer to the [Travis Documentation](https://docs.travis-ci.com/user/getting-started/)
 - GitLab CI: refer to the [GitLab CI Documentation](https://about.gitlab.com/gitlab-ci/)
 - Azure Pipelines: refer to the [Azure Pipelines Documentation](https://docs.microsoft.com/fr-fr/azure/devops/pipelines/?view=vsts)
+- Github CI
```
Comment (pascalgrimaud) [inline pages/setting_up_ci.md]: can you change to GitHub Actions ? In fact, it's not GitHub CI lol

File: pages/setting_up_ci.md [hunk 2]
```diff
@@ -43,6 +44,7 @@ The CI/CD pipeline you want to generate:
 - Jenkins pipeline
 - Azure Pipelines
 - GitLab CI
+- GitHub CI
```
Comment (pascalgrimaud) [inline pages/setting_up_ci.md]: same

**Comment by atomfrede** [comment]
I thought the same and looked at our ci prompt which uses GitHub CI too.
But I would also say let's use GitHub Actions

**Comment by pascalgrimaud** [comment]
Our prompt needs to be updated.

**Comment by deepu105** [comment]
We should 💯 do this

**Comment by atomfrede** [comment]
Adapted to the current prompts.

File: pages/bug-bounties.md [hunk 3]
```diff
@@ -49,7 +49,7 @@ In order to claim the money, you must:
 
 - Create a Pull Request that fixes a ticket with the "\$\$ bug-bounty \$\$" label.
 - In order to close the ticket automatically, you must have one commit message with the `Fix` keyword. For example, `Fix #1234` to close ticket `#1234`.
-- That Pull Request must be merged by someone from the core team. If there are several Pull Requests, the core team member either selects the most recent one or the best one - that's up to him to decide what is best for the project.
```
Comment (cbornet) [inline pages/bug-bounties.md]: Happy you found that one before Twitter 😄

**Comment by jdubois** [comment]
Really great work, thanks @atomfrede !! We have same kind of tool internally at Microsoft, I didn't know we could also benefit from this without paying!

**Comment by atomfrede** [comment]
@jdubois Sound good. I have started to work through the pages with alex which is really helpful for now. Will provide some more merge requests soon.

**Comment by cbornet** [comment]
@atomfrede can you also check the generated README ?

**Comment by atomfrede** [comment]
@cbornet Sure will do it this evening.

**Pull request merged**